lake/ztests/seek-index-null.yaml

```diff
@@ -5,7 +5,7 @@ script: |
     echo // $o | tee /dev/stderr
     zed create -q -seekstride 2KB -orderby ts:$o $o
     zed load -q -use $o babble.zson null.zson
-    zed query -z -s "from $o range 2020-04-21T23:59:26.063Z to 2020-04-21T23:59:38.069Z"
+    zed query -z -s -use $o "ts >= 2020-04-21T23:59:26.063Z and ts <= 2020-04-21T23:59:38.069Z"
```

Comment:
Any reason for changing from `from` to `-use`? (Just curious.)

Reply:
It's more a matter of taste. I don't like writing from in single pool queries and like the consistency of using the `-use` arg as with the other commands. Also with this change `from` isn't necessary.